### ISSUE TITLE
Error while using create-tables goal in conjunction with arguments (inMemory, ...)

### ISSUE THREAD

**Issue by sergiovm** [jcabi/jcabi-dynamodb-maven-plugin#47]
We are getting an error when usin create-tables goal and -inMemory, -sharedDb arguments.

This is our pom.xml:
````
(...)
<plugins>
    <plugin>
	<groupId>com.jcabi</groupId>
	<artifactId>jcabi-dynamodb-maven-plugin</artifactId>
	<version>0.7.1</version>
	<executions>
		<execution>
			<id>dynamodb-integration-test</id>
			<goals>
				<goal>start</goal>
				<goal>create-tables</goal>
				<goal>stop</goal>
			</goals>
			<configuration>
				<port>${dynamodblocal.port}</port>
				<dist>${project.build.directory}/dynamodb-dist</dist>
				<arguments>
					<argument>-inMemory</argument>
					<argument>-sharedDb</argument>
				</arguments>
				<tables>
					<table>${basedir}/src/test/resources/dynDB/TableA.json</table>
					<table>${basedir}/src/test/resources/dynDB/TableB.json</table>
					<table>${basedir}/src/test/resources/dynDB/TableC.json</table>
				</tables>
			</configuration>
		</execution>
	</executions>
    </plugin>
(...)
</plugins>
````

And the output we are getting is:
````
[INFO] #start(*****, 8000, 'C:\Program Files\Java\jdk1.8.0_66\jre', '[-inMemory, -sharedDb]'): in 75ms
[INFO]
[INFO] --- jcabi-dynamodb-maven-plugin:0.7.1:create-tables (dynamodb-integration-test-setup) @ project  ---
[INFO] >> Error while parsing options. Unrecognized option: -inMemory -sharedDb
[INFO] >> usage: java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar
[INFO] >>             [-port <port-no.>] [-inMemory] [-delayTransientStatuses]
[INFO] >>             [-dbPath <path>][-sharedDb] [-cors <allow-list>]
[INFO] >>  -cors <arg>                Enable CORS support for javascript against a
[INFO] >>                             specific allow-list list the domains separated
[INFO] >>                             by , use '*' for public access (default is
[INFO] >>                             '*')
[INFO] >>  -dbPath <path>             Specify the location of your database file.
[INFO] >>                             Default is the current directory.
[INFO] >>  -delayTransientStatuses    When specified, DynamoDB Local will introduce
[INFO] >>                             delays to hold various transient table and
[INFO] >>                             index statuses so that it simulates actual
[INFO] >>                             service more closely. Currently works only for
[INFO] >>                             CREATING and DELETING online index statuses.
[INFO] >>  -help                      Display DynamoDB Local usage and options.
[INFO] >>  -inMemory                  When specified, DynamoDB Local will run in
[INFO] >>                             memory.
[INFO] >>  -optimizeDbBeforeStartup   Optimize the underlying backing store database
[INFO] >>                             tables before starting up the server
[INFO] >>  -port <port-no.>           Specify a port number. Default is 8000
[INFO] >>  -sharedDb                  When specified, DynamoDB Local will use a
[INFO] >>                             single database instead of separate databases
[INFO] >>                             for each credential and region. As a result,
[INFO] >>                             all clients will interact with the same set of
[INFO] >>                             tables, regardless of their region and
[INFO] >>                             credential configuration. (Useful for
[INFO] >>                             interacting with Local through the JS Shell in
[INFO] >>                             addition to other SDKs)
[INFO] >> Error processing the supplied command line arguments
````
It seems dynamodb is starting properly but then fails in table creation goal due to the arguments.

**Comment by dmarkov** [comment]
@yegor256 please pay attention to this issue ([par.21](http://at.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@rultor release, tag is `0.7.2`

**Comment by yegor256** [comment]
@sergiovm this is very strange, since all our tests pass without issues, both on Linux and Windows. can you please try version 0.7.2?

**Comment by sergiovm** [comment]
The issue is fixed in version 0.7.2! Thanks a lot!
